core/trino-spi/src/main/java/io/trino/spi/security/AccessDeniedException.java

```diff
@@ -273,6 +273,11 @@ public static void denyShowColumns(String tableName)
         throw new AccessDeniedException(format("Cannot show columns of table %s", tableName));
     }
 
+    public static void denyShowColumns(String tableName, String extraInfo)
```

Comment:
This is out of the scope of this commit

Reply:
I changed the title of the PR to include both overloadings.